uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-month.component.js

```diff
@@ -0,0 +1,485 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/moment',
+  'leave-absences/shared/modules/components'
+], function (_, moment, components) {
+  components.component('leaveCalendarMonth', {
+    bindings: {
+      contacts: '<',
+      month: '<',
+      period: '<',
+      showContactName: '<',
+      showOnlyWithLeaveRequests: '<',
+      supportData: '<'
+    },
+    templateUrl: ['shared-settings', function (sharedSettings) {
+      return sharedSettings.sharedPathTpl + 'components/leave-calendar-month.html';
+    }],
+    controllerAs: 'month',
+    controller: ['$log', '$q', '$rootScope', 'Calendar', 'LeaveRequest', 'shared-settings', controller]
+  });
+
+  function controller ($log, $q, $rootScope, Calendar, LeaveRequest, sharedSettings) {
+    $log.debug('Component: leave-calendar-month');
+
+    var dataLoaded = false;
+    var calendars = {};
+    var leaveRequests = {};
+
+    var vm = this;
+    vm.currentPage = 0;
+    vm.pageSize = 20;
+    vm.visible = false;
```

Comment:
well it means that the month is visible 
```html
<section class="panel panel-default" ng-if="month.visible">
```
you have any suggestion on how to make it more meaningful?